.github/workflows/code-formatting.yml

```diff
@@ -0,0 +1,37 @@
+name: Codeformatting
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  autoblack:
+    if: "!startsWith(github.event.head_commit.message, 'chore') && !startsWith(github.event.head_commit.message, 'build: hotfix')"
```

Comment:
minor thing, but `hotfix` could also be formatted right?

Reply:
I think the idea is to not interfere with hot fixes too much. It is the same behavior as for other github actions. Thus, I'd keep it in sync.

Reply:
most of the `hotfix` is about releasing new versions.